app/predictorScores/index.njk

```diff
@@ -61,16 +61,21 @@
             </div>
             <div class="govuk-grid-row">
                 <div class="govuk-grid-column-one-half">
+                    {% if predictorScores.current.scores.OSPC.level !== 'NOT_AVAILABLE' %}
```

Comment:
would do you still need to show a component even if is empty?

![image](https://user-images.githubusercontent.com/9938931/129869251-ff448d76-87c8-4514-8a60-279db5c5aab6.png)